src/main/java/frc/robot/commands/LiftDriveForward.java

```diff
@@ -0,0 +1,49 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+
+public class LiftDriveForward extends Command {
+  public LiftDriveForward() {
+    requires(Robot.liftSystem);
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+  }
+
+  // Called just before this Command runs the first time
+  @Override
+  protected void initialize() {
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+    Robot.liftSystem.driveForward();
+    
+  }
+
+  // Make this return true when this Command no longer needs to run execute()
+  @Override
+  protected boolean isFinished() {
```

Comment:
We thought about using a limit switch, but putting one on the front climbing mechanism is hard because we don't already have wires going down there (no motors). Will and I decided to use a Timer

Reply:
Is the mechanism hitting a physical stop? You can detect the current spike when the motors stall out and use that as your sensor

Reply:
The gear ratio is very high -- 252:1. I am concerned that the wheels will just start slipping before a spike in current is detected that is large enough to be separated out from the nois

Reply:
Can the operator hold a button until the end condition is reached?